cursorless-talon/src/spoken_forms.json

```diff
@@ -0,0 +1,260 @@
+{
+  "actions": {
+    "simple_action": {
+      "bottom": "scrollToBottom",
```

Comment:
I'd be tempted to map the other direction (id to spoken form), as that's how we prefer to represent things typescript-side, but again, maybe a reasonable first step to keep things closer to how they are now?

Reply:
I'd also be tempted to map to a list of spoken forms, and to allow richer data, such as a default spoken form that is disabled by default, without using shorthand such as `-foo`, but again, maybe for further work

Reply:
Kiss for now :D. Format change could be reasonable when we do meta updater.